source/docs/software/getting-started/options-for-programming.rst

```diff
@@ -28,9 +28,10 @@ Disadvantages
 - Not recommended if you're already familiar with some type of programming
 
   - Primarily a teaching tool for people with no programming experience
+- Using external libraries is impossible
```

Comment:
```suggestion
- Using external libraries is significantly more difficult than with other options
```
I don’t think the original statement is true? At the very least, there are provisions for libraries to provide custom blocks; I am pretty sure you can upload libraries to the management dashboard for use in onbot and that should allow their use in blocks as well.

Reply:
I don't know of a single library that supports this, even if it's theoretically possible.